cobbler/tftpgen.py

```diff
@@ -1072,6 +1072,12 @@ def generate_bootcfg(self, what, name):
 
         blended = utils.blender(self.api, False, obj)
 
+        if distro.os_version.startswith("esxi"):
+            realbootcfg = open(os.path.join(os.path.dirname(distro.kernel), 'boot.cfg')).read()
```

Comment:
Please close the file again. Since you are not using the `with` syntax, this needs to be done manually via `.close()`